### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by yurii-krymskiy**
- [DEMO LINK](https://yurii-krymskiy.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://yurii-krymskiy.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,31 @@
...
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#home" class="logo"><img src="/images/logo.png" alt="logo"></a>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:18]:
```suggestion
      <a href="#home" class="logo">
        <img src="/images/logo.png" alt="logo">
      </a>
```

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,31 @@
...
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link"><a class="link is-active" href="#">Apple</a></li>
+          <li class="nav__link"><a class="link" href="#">Samsung</a></li>
+          <li class="nav__link"><a class="link" href="#">Smartphones</a></li>
+          <li class="nav__link"><a class="link computer__link" data-qa="hover" href="#">Laptops & computers</a></li>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:25]:
```suggestion
          <li class="nav__link">
            <a class="link" href="#">Smartphones</a>
          </li>
          
          <li class="nav__link">
            <a 
              class="link computer__link" 
              data-qa="hover" 
              href="#"
            >Laptops & computers</a>
          </li>
```
make empty line between siblings.
new elevent write on new line, if tag have three or more attributes - place each on new line
check code style in other places in html

File: src/style.css [hunk 3]
```diff
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+}
```
Comment (VitaliyBondarenko1982) [inline src/style.css:6]: move font-family and font-size to html selector

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,31 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#home" class="logo"><img src="/images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link"><a class="link is-active" href="#">Apple</a></li>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:22]:
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
```

make more meaningful class names.

File: src/style.css [hunk 5]
```diff
@@ -1 +1,62 @@
...
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  list-style-type: none;
+  margin-right: 50px;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:25]: instead making margin-right for nav list make horizontal paddings for header
pay attention on reference. list shouldn't have any horizontal spaces
Screenshot from 2023-04-11 14-36-10

File: src/style.css [hunk 6]
```diff
@@ -1 +1,62 @@
...
+}
+
+.nav__link {
+  margin-right: 20px;
+}
+
+.link {
+  text-decoration: none;
+  color: #000;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:34]: pay attention on reference and compare with test
link should be on full height of header
add vertical paddings for fix it
Screenshot from 2023-04-11 14-40-01

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
all tests failed.
try to fix them.
some comments can help to do it.

File: src/style.css (hunk 6 above)
Comment (yurii-krymskiy) [reply src/style.css:34]: Should I add vertical paddings for class="computer__link" or for header ? there is mockup 22 and 23 so horizontal should be 0 0 ?

**Pull request closed without merging**